### PR TITLE
Feature/dom-manipulation-ET

### PR DESCRIPTION
**Pull request by ectweitmann**
## What Features are Being Added?
- Added Home button functionality

## What is Being Refactored?
- The filtering by tags functionality
- Removed bugs with big modal view of recipes

## What Do You Need From Your Teammates?
- Please review my additions and let me know any of  any revisions I need to make. 

## Any Other Notes/Questions/Problems?
- Nope!

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -130,20 +146,20 @@ const createRecipeCardEventListener = () => {
...
 const displayBigModal = (event) => {
-  removeClass([bigModal], 'hidden');
+  removeClass([bigModal, homeButton], 'hidden');
   addClass([mainDisplay], 'hidden');
   populateBigModal(event);
 }
 
 const populateBigModal = (event) => {
-  let selectedRecipe = cookbook.currentRecipes.find(recipe => recipe.id === parseInt(event.target.parentNode.id));
+  let selectedRecipe = cookbook.currentRecipes.find(recipe => recipe.id === parseInt(event.target.id));
```
Comment (alexmfritz) [inline src/scripts.js:162]: Good catch recognizing the event.target.id was not coming out as the same data type and needed to be parsed

File: src/scripts.js [hunk 2]
```diff
@@ -160,6 +176,14 @@ window.addEventListener('load', displayCurrentRecipes);
 
 filterButton.addEventListener('click', clickFilterButton);
 
+homeButton.addEventListener('click', clickHomeButton);
```
Comment (alexmfritz) [inline src/scripts.js:179]: I love the difference in event listener functionality between the home button, filter button being unclicked and the logo refreshing the page.

File: src/scripts.js [hunk 3]
```diff
@@ -53,51 +53,67 @@ const displayCurrentRecipes = () => {
...
+        <input class="filter" type="checkbox" name="${tag}" id ="${tag}" />
+        <label for="${tag}">${tag}</label>
+      </section>
+    </section>`
   });
 }
 
 const getFilterTags = () => {
-  return cookbook.currentRecipes.reduce((accumulator, recipe) => {
+  return cookbook.recipes.reduce((accumulator, recipe) => {
```
Comment (Daniel-OC) [inline src/scripts.js:116]: Smart solve here!

File: src/styles.css [hunk 4]
```diff
@@ -239,7 +235,7 @@ article button {
 }
 
 .orange {
-	border-color: #FF9F1C;
+	background-color: #FF9F1C;
```
Comment (alexmfritz) [inline src/styles.css:238]: Please, please, please change these colors

**Pull request merged**